src/web/components/Avatar.tsx

```diff
@@ -1,35 +1,28 @@
 import React from 'react';
 import { css, cx } from 'emotion';
 
-import { Pillar } from '@guardian/types';
-import type { Format } from '@guardian/types';
-
-import { pillarPalette } from '@frontend/lib/pillars';
-
 const contributorImage = css`
 	border-radius: 100%;
 	object-fit: cover;
 	height: 100%;
 	width: 100%;
 `;
 
-const pillarBackground = (pillar: Theme = Pillar.Opinion) =>
+const backgroundStyles = (palette: Palette) =>
 	css`
-		background-color: ${pillar === Pillar.Opinion
-			? pillarPalette[pillar].main
-			: pillarPalette[pillar].bright};
+		background-color: ${palette.background.avatar};
 	`;
 
 export const Avatar: React.FC<{
 	imageSrc: string;
 	imageAlt: string;
-	format: Format;
-}> = ({ imageSrc, imageAlt, format }) => {
+	palette: Palette;
```

Comment:
This _replaces_ `format` with `palette` because we reached the higher plain where format is no longer needed and all choices we contained inside `palette` 🧘